migrations/Version20241029125655.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Surfnet\ServiceProviderDashboard\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241029125655 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE service ADD invite_role_id INT DEFAULT NULL, DROP invite_urn');
```

Comment:
The DROP portion of the migrations `up` query is probably not necessary right?

Reply:
Ah, good catch!